docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -312,52 +255,25 @@ keycloak.init({
-side. Affected is for example Chrome starting with
-version 84.
+All features are supported if SSL / TLS connection is configured on the {project_name} side as well as on the application side. Affected is for example Chrome starting with version 84.
 
 ===== Browsers with Blocked Third-Party Cookies
-Session Status iframe is not supported and is automatically disabled if such browser behavior is detected by the JS adapter.
-This means the adapter cannot use session cookie for Single Sign-Out detection and have to rely purely on tokens. This
-implies that when user logs out in another window, the application using JavaScript adapter won't be logged out until it
-tries to refresh the Access Token. Therefore, it is recommended to set Access Token Lifespan to relatively short time, so
-that the logout is detected rather sooner than later. Please see link:{adminguide_link}#_timeouts[Session and Token Timeouts].
+Session Status iframe is not supported and is automatically disabled if such browser behavior is detected by the adapter. This means the adapter cannot use session cookie for Single Sign-Out detection and have to rely purely on tokens. This implies that when user logs out in another window, the application using the adapter won't be logged out until it tries to refresh the Access Token. Therefore, it is recommended to set Access Token Lifespan to relatively short time, so that the logout is detected rather sooner than later. Please see link:{adminguide_link}#_timeouts[Session and Token Timeouts].
 
-_Silent_ `check-sso` is not supported and falls back to regular (non-silent) `check-sso` by default. This behaviour can
-be changed by setting `silentCheckSsoFallback: false` in the options passed to the `init` method. In this case, `check-sso`
-will be completely disabled if restrictive browser behavior is detected.
+_Silent_ `check-sso` is not supported and falls back to regular (non-silent) `check-sso` by default. This behavior can be changed by setting `silentCheckSsoFallback: false` in the options passed to the `init` method. In this case, `check-sso` will be completely disabled if restrictive browser behavior is detected.
 
-Regular `check-sso` is affected as well. Since Session Status iframe is unsupported, an additional redirect to {project_name}
-has to be made when the adapter is initialized to check user's login status. This is different from standard behavior when
-the iframe is used to tell whether the user is logged in, and the redirect is performed only when logged out.
+Regular `check-sso` is affected as well. Since Session Status iframe is unsupported, an additional redirect to {project_name} has to be made when the adapter is initialized to check user's login status. This is different from standard behavior when the iframe is used to tell whether the user is logged in, and the redirect is performed only when logged out.
```

Comment:
```suggestion
Regular `check-sso` is affected as well. Since Session Status iframe is unsupported, an additional redirect to {project_name} has to be made when the adapter is initialized to check the user's login status. This check is different from the standard behavior when the iframe is used to tell whether the user is logged in, and the redirect is performed only when the user is logged out.
```